docs/reference/tools-reference/contentsafety_text_tool.md

```diff
@@ -0,0 +1,53 @@
+# Content Safety (Text)
+
+Azure Content Safety is a content moderation service developed by Microsoft that help users detect harmful content from different modalities and languages. This tool is a wrapper for the Azure Content Safety Text API, which allows you to detect text content and get moderation results. See the [Azure Content Safety](https://aka.ms/acs-doc) for more information.
+
+## Requirements
+
+- For AzureML users, the tool is installed in default image, you can use the tool without extra installation.
+- For local users,
+  `pip install promptflow-tools`
+  
```

Comment:
> [!NOTE]
> Content Safety (Text) tool is now incorporated into the latest `promptflow-tools` package. If you have previously installed the package `promptflow-contentsafety`, please uninstall it to avoid the duplication in your tool list.